### ISSUE TITLE
404 route when viewing logs

### ISSUE THREAD

**Issue by XPLOT1ON** [drone/drone#2182]
I am getting 404 when trying to click expand log window during testing. This only happens during the testing stage. once the entire run is complete, I was able to view the results without error.

`drone-server_1  | time="2017-08-26T18:55:11Z" level=error msg="Error #01: sql: no rows in result set\n" ip=<PUBLIC_IP> latency=2.112947ms method=GET path="/api/repos/<OWNER>/<REPO>/logs/2/1/backend" status=404 time="2017-08-26T18:55:11Z" user-agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.101 Safari/537.36"`

I can confirm the backend did found and initialise the route
`drone-server_1  | [GIN-debug] GET    /api/repos/:owner/:name/logs/:number/:ppid/:proc --> github.com/drone/drone/server.GetBuildLogs (16 handlers)`

**Comment by bradrydzewski** [comment]
duplicate of https://github.com/drone/drone-ui/issues/144
